src/cmp.test.ts

```diff
@@ -106,10 +106,38 @@ describe('cmp', () => {
                 expect(myCallBack).toBeCalledWith(iabNullState);
             });
 
+            //     it('executes advertisement callback each time consent nofication triggered with updated state', () => {
```

Comment:
Seems like this test case was testing an important piece of behaviour. Is this now covered elsewhere?

Reply:
Good spot, this test has been duplicated and updated, the commented out version here can be removed